framework/helpers/logger/logger.livecodescript

```diff
@@ -1,18 +1,53 @@
 script "Logger Library"
-constant kLogTypes = "developer,extensions,network,msg"
+constant kLogTypes = "developer,extensions,network,msg,error"
 
 local sLogTarget
 local sLogA
 local sNetworkFilters
+local sColDelim, sRowDelim
+local sIncludeLogTypeBool
 
 
 on libraryStack
   if the target is not me then pass libraryStack
 
-  put true into sLogA["developer"]
-  put false into sLogA["extensions"]
-  put false into sLogA["network"]
-  put false into sLogA["msg"]
+  #<Initial settings>
+    #<log types>
+      put "developer" into logtypes #default
+      if levureAppHasProperty("logger-types") then put levureAppGet ("logger-types") into logtypes
+      loggerSetTypes logtypes
+    #</log types>
+
+    #<log target>
+      put empty into logTarget #default, i.e. ignore log messages
+      if levureAppHasProperty("logger-target") then put levureAppGet ("logger-target") into logTarget
+      loggerSetTarget logTarget
+    #<log target>
+
+    #<column delimiter>
+      put tab into colDelim #default
+      /*
+      Future use once issue 102 is resolved.  Could have an app.yml setting (or a preference setting, or both), but instead of ascii characters, could use something like "<HT>" or a list of ascii codes e.g. "9"
+      if levureAppHasProperty("logger-column-delimiter") then put levureAppGet ("logger-column-delimiter") into colDelim
+      */
```

Comment:
This is a place holder for the future.  What I would like to do (either now or soon) is give the developer the ability to set the delimiters in the app.yml file, too.  Before I do that, I want to finish issue 102, which is about building sanity checking into helper.yml - so that if the developer hoses something when customizing the settings for the helper, they don't break it.

Reply:
But you are using the configuration file just above? Why wait on this one?

Reply:
Just because I wanted to implement the sanity checking of .yml files first.  No worries, I'll add it now.